js/app.js

```diff
@@ -0,0 +1,106 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav-menu');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
+document.querySelectorAll('.a-links').forEach((n) => n
+  .addEventListener('click', () => {
+    hamburger.classList.remove('active');
+    navMenu.classList.remove('active');
+  }));
+
+const featuredSpeakers = document.getElementById('featuredSpeakers');
+
+const allSpeakers = [
+  {
+    image: './assets/images/speaker_012.png',
+    speakerName: 'Kim Doctor',
+    profession: 'Berkman Professor Shadowed physicians in multiple departments',
+    carreer: 'Current Physician Assistant (PA) student',
+  },
+
+  {
+    image: './assets/images/speaker_012.png',
+    speakerName: 'osugo Benkler',
+    profession: 'Professor of Entrepreneurial Business Studies at Harvard Law School',
+    carreer: 'Jomo kenyatta University',
+  },
+
+  {
+    image: './assets/images/speaker_012.png',
+    speakerName: 'Joes Njoroge',
+    profession: 'Software Engineer at microsoft Studies at Harvard Law pauls college',
+    carreer: 'Jes studies commons-based communications book',
+  },
+
+  {
+    image: './assets/images/speaker_012.png',
+    speakerName: 'Kimbarly Beverly',
+    profession: 'Lecture of Entrepreneurial Business Studies at Harvard Law Sch School',
+    carreer: 'commons-based peer production, and published his seminal book',
+  },
+
+  {
+    image: './assets/images/speaker_012.png',
+    speakerName: 'Grace Benkler',
+    profession: 'Grace Professor of Entrepreneurial Legal Studies at Harvard Law School',
+    carreer: 'Grace studies commons-based peer production, and published his seminal book',
+  },
+
+  {
+    image: './assets/images/speaker_012.png',
+    speakerName: 'Truphena Benkler',
+    profession: 'Lecture of Entrepreneurial Business Studies at Harvard Law Sch School',
+    carreer: 'Truh studies commons-based computer science and programmings',
+  },
+];
+
+const readMore = document.querySelector('.readMore');
+
+window.addEventListener('load', (() => {
+  function addfeatured() {
+    let addfeatureSpeakers = '';
+    allSpeakers.forEach((speaker, index) => {
+      if ((window.screen.availWidth >= 768)) {
+        addfeatureSpeakers += `<div class="speakers-card">
+      <img src="${speaker.image}" alt="image1"/>
```

Comment:
- [ ]  As said previously, Kindly make sure you use the right path for your images, for a better user experience, the image should be visible on your page and not broken.
`Hint: As you may notice, your app.js file is not in the root of the directory, so to access the image in the assets folder, you may consider using ../assets and not ./`
![image](https://user-images.githubusercontent.com/97100091/226142983-8d82fddf-ddf5-4e75-aeea-d9c4f9dcf925.png)

Reply:
Sorted. Due to time i linked an image. I will work more on it